books-list.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet"  type="text/css" href="application.css">
+  </head>
+  <body>
+    <div class="list-container">
+      <div class="book-item">
+        <a href="index.html" class="book-cover-mini"></a>
```

Comment:
Here, use the image tag inside the anchor, think about this: in a real case, you are goin to get the imageUrl from an endpoint and map the response here with javascript or in this case with angular, so all images aren't going to be the same.